src/components/accomodations_card/index.js

```diff
@@ -0,0 +1,30 @@
+import EmojiPicker from "emoji-picker-react";
+
+const AccomodationsCard = () => {
+    return (
+        <div className='accomodations-card'>
+            <div className="accomodations-card">
+                <p><h3>Hotel Accomodations </h3></p>
```

Comment:
You don't need to have the `<p>` tag on wrapping the header tags. both are block level elements and some browsers throw errors when you wrap a "lower level" block element around a higher one